mindsdb/api/mysql/mysql_proxy/executor/data_types.py

```diff
@@ -0,0 +1,33 @@
+
+from typing import List, Dict
+class ANSWER_TYPE:
+    __slots__ = ()
+    TABLE = 'table'
+    OK = 'ok'
+    ERROR = 'error'
+
+
+ANSWER_TYPE = ANSWER_TYPE()
+
+
+class ExecuteAnswer:
```

Comment:
Is it using anywhere? why we need another copy of SQLAnswer?

Reply:
ExecuteAnswer is used only inside executor, to pass result from command_executor. It is created for replacing SQLAnswer to do not change logic during current merge. I think in the future we have to refactor executor_commands module from one big class to several related to commands and also do this module simpler.
SQLAnswer is used only to pass result to MysqlProxy.